myhpi/core/migrations/0002_rootpage.py

```diff
@@ -50,7 +50,7 @@ def create_homepage(apps, schema_editor):
     Footer.objects.get_or_create(
         column_1="# Fachschaft\r\n\r\n- [Twitter](https://twitter.com/fachschaftsrat)\r\n- [Discord](https://discord.com)\r\n- [Telegram](https://telegram.org)",
         column_2="# Rechtliches\r\n\r\n- [Impressum]()\r\n- [Datenschutzerklärung]()",
-        column_3="# Entwicklung\r\n\r\n- [GitHub](https://github.com/fsr-de/myHPI/)",
+        column_3="# Entwicklung\r\n\r\n- [GitHub](https://github.com/fsr-de/myHPI/)\r\n[MYHPI-VERSION]",
```

Comment:
maybe we should re-visit how the footer is built. I already removed the last column for the language as this is handled programatically, now we have this variable here where the preprocessor seems like a very complicated way to achieve this. But not part of this PR

Reply:
#455